js/script.js

```diff
@@ -2,20 +2,95 @@ const mainMenu = document.querySelector('.mainMenu');
+  diplayUl: ['|', 'html', '|', 'Ruby on rails', '|', 'css', '|', 'Github'],
+}];
+
+const displayWork = document.getElementById('displayWork');
+
+mobileModalContent.displayImgSmall = './images/Snapshoot-Portfolio.png';
+mobileModalContent.descriptionMobile = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.';
+mobileModalContent.diplayUl = ['|', 'html', '|', 'Ruby on rails', '|', 'css', '|'];
+desktopModalContent.title = 'Multi-post stories';
+desktopModalContent.closeWork = '<i class="closeWork fa fa-times"></i>';
+desktopModalContent.displayImgLarge = './images/Snapshoot-Portfolio-lg.png';
+desktopModalContent.descriptionDesktop = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.';
+desktopModalContent.diplayUl = ['|   ', 'html   ', '|   ', 'Ruby on rails   ', '|   ', 'css   ', '|   ', 'Github   ', '|   '];
+
+const langArray = mobileModalContent.diplayUl.join('  ');
+const langArrayLg = desktopModalContent.diplayUl.join('  ');
+
+displayWork.innerHTML = `<div class="headerDis"><h2 class="displayHeader">${desktopModalContent.title}</h2><span> <a onclick="closePopupModal()" href="" class="closeWork">${desktopModalContent.closeWork}</a></span></div>
+<img class="displayImg large" src="${desktopModalContent.displayImgLarge}" alt="Project Image on desktop device">
+<img class="displayImg small" src="${mobileModalContent.displayImgSmall}" alt="Project Image on mobile device">
+<p class="displayP">${desktopModalContent.descriptionDesktop}</p>
+<ul class="diplayUl">${langArray}</ul>
+<ul class="diplayUlLG">${langArrayLg}</ul>
+<div class="footerDis">
+<button class="buttonDis buttonOne" type="button"><a href="https://abigiyaty.github.io/Portofolio-/" target="_blank">See Live <i class="disFA fa fa-chevron-right"></i></a></button>
+<button class="buttonDis buttonTwo" type="button"><a href="https://github.com/AbigiyaTY/Portofolio-" target="_blank">See Source <i class="disFA fa fa-github"></i></a></button>
+</div>`;
+
```

Comment:
- [ ] Kindly make sure you implement **an array of objects** to store JavaScript objects containing the information for each project. And each object contains at least the following information about the project: `name`, `description`, `featured image`, `technologies`,` link to live version`, and `link to the project source code`. i.e.:
```javascript
const projects = [
 {
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    source_link: "link_to_live_version.com",
    live_link: "link_to_live_version.com",
},
...
]
```

- [ ] Also make sure the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **i.e. As I mentioned in the previous comment HTML code for the projects list should be removed, and instead it should be added via JavaScript.**